CHANGELOG-1.1.md

```diff
@@ -1,3 +1,39 @@
+# TiDB Operator v1.1.3 Release Notes
+
+## Action Required
+
+- Add a field `cleanData` in `BackupSpec` to denote whether to clean backup data before the object is deleted from the cluster (defaults to false). If you want to clean backup data as before, please set `spec.cleanData` in `Backup` yaml or `spec.backupTemplate.cleanData` in `BackupSchedule` yaml to true. ([#2892](https://github.com/pingcap/tidb-operator/pull/2892), [@lichunzhu](https://github.com/lichunzhu))
```

Comment:
- Add a field `cleanPolicy` in `BackupSpec` to denote clean policy for backup data before the object is deleted from the cluster (defaults to `Retain`). If you want to clean backup data as before, please set `spec.cleanPolicy` in `Backup` yaml or `spec.backupTemplate.cleanPolicy` in `BackupSchedule` yaml to `Delete`. ([#3002](https://github.com/pingcap/tidb-operator/pull/3002), [@lichunzhu](https://github.com/lichunzhu))

Reply:
@cofyc @DanielZhangQD PTAL